### PR TITLE
Make AssessmentViewModel open to be able to override BranchNodeStateI…

### PR DESCRIPTION
**Pull request by elkasvirElka**
…mpl field

### CODE REVIEW

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt [hunk 1]
```diff
@@ -40,45 +40,57 @@ open class AssessmentFragment : Fragment() {
...
+            packageName = packageName
+        )
+
+        val assessmentProvider =
+            FileAssessmentProvider(fileLoader, assessmentGroup, getJsonLoader())
+        viewModel = ViewModelProvider(
+            this, AssessmentViewModelFactory()
+                .create(assessmentId, assessmentProvider)
+        )
+            .get(AssessmentViewModel::class.java)
```
Comment (nategbrown9) [inline presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt:84]: Do you want this pulled out into a helper method you can override?  Then you have less duplicate code than overriding initAssessmentViewModel().

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentViewModel.kt [hunk 2]
```diff
@@ -7,8 +7,12 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import org.sagebionetworks.assessmentmodel.*
 import org.sagebionetworks.assessmentmodel.navigation.*
+import org.sagebionetworks.assessmentmodel.presentation.mtbAssesssmentTest.MtbAssessmentViewModel
```
Comment (nategbrown9) [inline presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentViewModel.kt:10]: This import shouldn't be here.

**Review by nategbrown9 — CHANGES_REQUESTED** [review]
Looks good. Just need to remove an import.

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentViewModel.kt (hunk 2 above)
Comment (elkasvirElka) [reply presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentViewModel.kt]: deleted.

**Review by nategbrown9 — APPROVED** [review]

**Pull request merged**